adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/model/DatasourceProtocol.java

```diff
@@ -19,7 +19,7 @@
   private Protocol type;
 
   @NotNull
-  @Column(name = "protocol_parameters")
+  @Column(name = "protocol_parameters", length = 2000)
```

Comment:
I think we should not use magic numbers here. Please extract to a constant.

Reply:
Resolved by not having a maximum length anymore.